### PR TITLE
Adds Penguins, and Penguins to Winter Wonderland

### PR DESCRIPTION
**Pull request by LanCartwright**
:cl: optional name here
add: Penguins, penguin chicks, penguin eggs.
add: Penguin family to Winter Wonderland.
/:cl:

[why]: # A first step to OWS.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/penguin.dm [hunk 1]
```diff
@@ -0,0 +1,37 @@
+//Penguins
+
+/mob/living/simple_animal/pet/penguin
+	response_help  = "pets"
+	response_disarm = "bops"
+	response_harm   = "kicks"
+	speak = list("Gah Gah!", "NOOT NOOT!", "NOOT!", "Noot", "noot", "Prah!", "Grah!")
+	speak_emote = list("squawks", "gakkers")
+	emote_hear = list("squawk!", "gakkers!", "noots.","NOOTS!")
```
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/friendly/penguin.dm]: I think this would result in stuff like "penguin squawk!."

**Comment by WJohn** [comment]
did you forget to use the map merger?

**Comment by LanCartwright** [comment]
what is a map merger

**Comment by ExcessiveUseOfCobblestone** [comment]
lol

**Comment by PKPenguin321** [comment]
aw hell yeah

**Comment by Chocobro** [comment]
<J_Madison> oranges I have an excellent idea
<J_Madison> I'm gonna bypass the map merger entirely
<J_Madison> by adding the spawners via notepad++

**Comment by LanCartwright** [comment]
image

**Comment by WJohn** [comment]
These actually look pretty cute. Nice work.

**Comment by LanCartwright** [comment]
NEXT SPRITING PROJECT: DUCKS
image

**Comment by Repukan** [comment]
Do the penguins come with an anime cutout?

**Comment by dannno** [comment]
thanks wjohn

**Comment by swindly** [comment]
Change minbodytemp?

**Comment by MrPerson** [comment]
This is a cute way to celebrate Pittsburgh winning the Stanley Cup.

File: code/modules/mob/living/simple_animal/friendly/penguin.dm [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	icon = 'icons/mob/penguins.dmi'
+
+/mob/living/simple_animal/pet/penguin/emperor
+	name = "Emperor penguin"
+	real_name = "penguin"
+	desc = "Emperor of all he surveys."
+	icon_state = "penguin"
+	icon_living = "penguin"
+	icon_dead = "penguin_dead"
+	butcher_results = list("")
```
Comment (lzimann) [inline code/modules/mob/living/simple_animal/friendly/penguin.dm]: remove the `""`

File: code/modules/mob/living/simple_animal/friendly/penguin.dm [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+/mob/living/simple_animal/pet/penguin/baby
+	speak = list("gah", "noot noot", "noot!", "noot", "squeee!", "noo!")
+	name = "Penguin chick"
+	real_name = "penguin"
+	desc = "Can't fly and can barely waddles, but the prince of all chicks."
+	icon_state = "penguin_baby"
+	icon_living = "penguin_baby"
+	icon_dead = "penguin_baby_dead"
+	density = 0
```
Comment (lzimann) [inline code/modules/mob/living/simple_animal/friendly/penguin.dm]: `FALSE`

File: code/modules/mob/living/simple_animal/friendly/penguin.dm (hunk 1 above)
Comment (lzimann) [inline code/modules/mob/living/simple_animal/friendly/penguin.dm]: It won't, all of the `emote_hear` has a `.` in the end. Should be fine.

**Pull request merged**